src/Microsoft.AspNetCore.Sockets/Transports/LongPollingTransport.cs

```diff
@@ -35,7 +34,10 @@ public LongPollingTransport(IReadableChannel<Message> connection, ILoggerFactory
 
             try
             {
-                using (var message = await _connection.ReadAsync(_cancellationSource.Token))
+                // TODO: We need the ability to yield the connection without completing the channel.
```

Comment:
I assume this will be used to timeout poll requests for which there was no data to send within a timeout so that the poll request are not killed by IIS/routers as stale requests. 

Do we want to open a bug for this?